e2e/testsuite/testsuite.go

```diff
@@ -258,12 +258,28 @@ func (s *E2ETestSuite) CreatePath(
 	return channelsA[len(channelsA)-1], channelsB[len(channelsB)-1]
 }
 
+type ChainChannelPair struct {
+	ChainIdx   int
+	ChannelIdx int
+}
+
```

Comment:
This might bloat a bit the file, but I'm not a fan of passing magic numbers around (it would be easy to switch them by mistake)